plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/Module.kt

```diff
@@ -33,6 +33,10 @@ internal sealed class Module {
     }
 
     class Java(private val convention: JavaPluginConvention) : Module() {
+        override val bootClasspath: Collection<File>
+            get() = File(System.getProperty("java.home")).walkTopDown()
+                .toList()
+                .filter { it.exists() && it.name == "rt.jar" }
```

Comment:
Is the correct JAR always going to be named "rt.jar" on all platforms? Is there a downside to just including all JAR files in the boot classpath?

Reply:
Oh, ok. Was just about to respond. :)